Birthday-2024-Project/ScenePrefabs/UISoundPlayer.tscn

```diff
@@ -2,7 +2,7 @@
 
 [ext_resource type="Script" path="res://Scripts/UI/UISoundPlayer.gd" id="1_hm1sf"]
 [ext_resource type="AudioStream" uid="uid://bnggqbcd14e6h" path="res://Resources/Audio/SFX/ButtonPress.wav" id="2_82vd0"]
-[ext_resource type="AudioStream" uid="uid://d0h81xxvsa0v" path="res://Resources/Audio/SFX/ButtonHover.wav" id="3_fnrm5"]
+[ext_resource type="AudioStream" path="res://Resources/Audio/SFX/ButtonHover.wav" id="3_fnrm5"]
```

Comment:
Seems like Godot removed the uid on this scene prefab for the hover sfx. Maybe check if this throws a warning like the other time the uid was missing

Reply:
It had to be some hiccup from Godot. Running game again restored UID